test/initial_release/GHTestServer.d

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************************
+
+    Github Test Server class
```

Comment:
I guess at some point this file should go to `test/lib` or something to be reusable from other tests, right?

Reply:
Yes, as soon as  a second test will be added I plan to adjust the structure accordingly